templates/_base.html

```
@@ -0,0 +1,95 @@
+{% load static %}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)